### PR TITLE
Added Text-to-Image generation

### PR DESCRIPTION
**Pull request by jaidh01**
Closes: #129 

- **Title**:   Text-to-image generator
- **Your Name**: Jai Dhingra
- **Idenitfy yourself**:  GSSOC-extd Contributor, Hacktoberfest

### Describe the add-ons or changes you've made 📃

Added the Text-to-Image Generator which runs on a stable diffusion model.

Note: - 
            1. Recommend using GPU for faster generation.
            2. For the first time it will download some files like: - weights, pipeline, etc., to run the stable diffusion model. 

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/Contributing.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷

https://github.com/user-attachments/assets/24cc3d0a-4a79-45f1-a888-0eeb3e0ded84

### CODE REVIEW

File: src/apps/pages/programs/imageProgram.py
Comment (Avdhesh-Varshney) [inline src/apps/pages/programs/imageProgram.py:1]: Undo these changes

File: src/apps/pages/programs/ImageGenerators/textToImageGenerator.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            # Optionally, save the image
+            image.save('src/apps/pages/programs/ImageGenerators/generated_image.png')
+            st.success("Image generated successfully!")
+        except Exception as e:
+            st.error(f"An error occurred: {str(e)}")
+    else:
+        st.warning("Please enter a prompt first.")
+
+# Call the main function directly
+textToImageGenerator()
```
Comment (Avdhesh-Varshney) [inline src/apps/pages/programs/ImageGenerators/textToImageGenerator.py:58]: Don't the function in the file, It is automatically called by the jarvis

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: src/apps/pages/programs/ImageGenerators/textToImageGenerator.py (hunk 1 above)
Comment (jaidh01) [reply src/apps/pages/programs/ImageGenerators/textToImageGenerator.py]: I removed the function calling from the file.
You can now review it.

File: src/apps/pages/programs/imageProgram.py
Comment (Avdhesh-Varshney) [inline src/apps/pages/programs/imageProgram.py:1]: Undo the changes of this file

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: src/apps/pages/programs/imageProgram.py
Comment (jaidh01) [reply src/apps/pages/programs/imageProgram.py:1]: I added that because it was showing me the ‘multiple identical form’ with the same ‘key’, even though there was only a single form instance in my code. So, it just ignores that warning, making the user interface look cleaner.

Comment (jaidh01) [reply src/apps/pages/programs/imageProgram.py:1]: Hey @Avdhesh-Varshney, Do I still have to undo these changes?

Comment (Avdhesh-Varshney) [reply src/apps/pages/programs/imageProgram.py:1]: This is shown because of calling the function from your own file and this function is also called by the `imageProgram.py` file. So, this error is popped up.

**Comment by jaidh01** [comment]
Hi @Avdhesh-Varshney, thank you for the guidance. I have made all the changes in `textToImageGenerator.py` and undone all the changes in `imageProgram.py` as well

File: src/apps/pages/models/HealthCareModels/BrainTumorModel/brainTumorModel.py
Comment (Avdhesh-Varshney) [inline src/apps/pages/models/HealthCareModels/BrainTumorModel/brainTumorModel.py:1]: Why this file is committed here?

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Avdhesh-Varshney** [comment]
Re-raise another clear pr
